frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,311 @@
+/* eslint-disable camelcase */
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { SectionHeading } from '../../utils';
+import { SaveAsDefaultCheckbox, FormProps } from './alert-manager-receiver-forms';
+
+const SMTP_GLOBAL_FIELDS = [
+  'smtp_from',
+  'smtp_smarthost',
+  'smtp_hello',
+  'smtp_auth_username',
+  'smtp_auth_password',
+  'smtp_auth_identity',
+  'smtp_auth_secret',
+  'smtp_require_tls',
+];
+const GLOBAL_FIELDS = [...SMTP_GLOBAL_FIELDS]; //TODO follow up PR will add advanced fields
+
+const getGlobalValue = (globals, propName) => {
+  const fieldName = SMTP_GLOBAL_FIELDS.includes(propName)
+    ? propName
+    : propName.replace('smtp_', 'email_'); // 'email_...' are globals for advanced fields
+  return globals[fieldName];
+};
+
+export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormChange }) => {
+  // disable saveAsDefault if all SMTP form fields match global values
+  const disableSaveAsDefault = SMTP_GLOBAL_FIELDS.every(
+    (propName) => formValues[propName] === getGlobalValue(globals, propName),
+  );
+
+  return (
+    <div data-test-id="email-receiver-form">
+      <div className="form-group">
+        <label className="control-label co-required" htmlFor="emailTo">
+          To Address
+        </label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          aria-describedby="email-to-help"
+          id="emailTo"
```

Comment:
By convention, `id` should be kebab case, not camel case (here and below).

Reply:
fixed